tests/estimation/test_get_estimation.py

```diff
@@ -1,18 +1,14 @@
 """
 get_estimation.py::get_estimation
 
-We test all the benchmark_mediation estimators for a certain tolerance.
+It tests all the benchmark_mediation estimators for a certain tolerance.
 The test is skipped if estimator has not been implemented yet,
 i.e. all outputs are NaN or NotImplementedError is raised.
```

Comment:
ValueError rather than NotImplementedError, no?

Reply:
Indeed I didn't update the docstring

Reply:
updated, 1ca7d17